packages/riipen-ui/src/components/Radio.test.jsx

```diff
@@ -0,0 +1,152 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Radio from "./Radio";
+
+describe("<Radio>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Radio />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Radio />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Radio().type.defaultProps;
+
+      const wrapper = mount(<Radio />);
+
+      expect(wrapper.find("Radio").props().checked).toEqual(
+        defaultProps.checked
+      );
+      expect(wrapper.find("Radio").props().disabled).toEqual(
+        defaultProps.disabled
+      );
+    });
+  });
+
+  describe("checked prop", () => {
+    it("sets valid custom checked", () => {
+      const checked = true;
+
+      const wrapper = mount(<Radio checked={checked} />);
+
+      expect(
+        wrapper
+          .find("Radio")
+          .find("input")
+          .props().checked
+      ).toEqual(checked);
+    });
```

Comment:
I am guessing this is throwing the warning:

```
Warning: Failed prop type: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.
```

To make this go away you could do `<Radio checked={checked} readOnly />`.

What's happening is that if you have an HTML checkbox and you say "this checkbox is checked", you basically can never uncheck it, so it's telling you to pass "readOnly" to the checkbox so that the browser knows that this checkbox will never be unchecked.

Reply:
The warning was affecting every test, so I've adjusted every mount function